.github/workflows/scripts/linux_arm64/build_enzyme.sh

```diff
@@ -0,0 +1,30 @@
+set -e -x
+cd /catalyst 
+# Process args
+export GCC_VERSION=$1
+export PYTHON_VERSION=$2
+export PYTHON_SUBVERSION=$3
+export PYTHON_PACKAGE_NAME=$4
+# Install system dependencies
+dnf update -y 
+dnf install -y libzstd-devel gcc-toolset-${GCC_VERSION} ${PYTHON_PACKAGE_NAME} ${PYTHON_PACKAGE_NAME}-devel
+dnf clean all -y 
+# Make GCC the default compiler
+source /opt/rh/gcc-toolset-${GCC_VERSION}/enable -y 
+export C_COMPILER=/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin/gcc 
+export CXX_COMPILER=/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin/g++ 
+# Set the right Python interpreter
+rm -rf /usr/bin/python3
+ln -s /opt/_internal/cpython-${PYTHON_VERSION}.${PYTHON_SUBVERSION}/bin/python3 /usr/bin/python3 
+export PYTHON=/usr/bin/python3
```

Comment:
This can likely be replaced by `export PYTHON=$(which python${PYTHON_VERSION}.${PYTHON_SUBVERSION})`, but I'll leave that suggestion for a follow up

Reply:
I got some strange behavior when I was not making sure that the python binary at `/usr/bin` was not the intended one, even if the `PYTHON` env var was set. Somewhere in the compilation scripts `/usr/bin/python3` was being accessed hard-coded, ignoring the `PYTHON` env var, specially a testing time. That's the reason of this enforcement.